makiflow/tools/test_visualizer.py

```diff
@@ -40,4 +41,41 @@ def plot_test_values(test_values, legends, x_label, y_label, save_path=None):
             fig.show()
         plt.close(fig)
 
-        
\ No newline at end of file
+    @staticmethod
+    def plot_numpy_dist_obs(values, legends, save_path=None, axes_x=None, axes_y=None):
+        """
+        Parameters
+        ----------
+        values : list
+            Set of values that need to plot.
+        legends : list
+            List of values names.
+        save_path : str
+            Path to save plotted figure.
+        axes_x : tuple
+            Tuple of size 2, example (min_x, max_x), where min_x - minimum on the axes x, max_x - maximum of the axes x.
```

Comment:
"Tuple of (min, max), where `max` and `min` are the maximum and minimum values on the X axis accordingly. If set to None, maximum and minimum values are not constrained."
The same with the Y axis.